### ISSUE TITLE
[ter-indent] Call expression arguments false positives

### ISSUE THREAD

**Issue by nathanmarks** [buzinas/tslint-eslint-rules#254]
With the following rule configuration:

```
    "ter-indent": [
      true,
      2,
      {
        "SwitchCase": 1,
        "MemberExpression": 1,
        "FunctionDeclaration": {
          "parameters": 1
        },
        "FunctionExpression": {
          "parameters": 1
        },
        "CallExpression": {
          "arguments": 1
        }
      }
    ],
```

This is reporting as a false positive:

```ts
  foo = this.actions$
    .ofType(
      'foo',
      'bar'
    );
```
It wants me to do this instead:

```ts
  foo = this.actions$
    .ofType(
    'foo',
    'bar'
    );
```

**Comment by rokerkony** [comment]
Hi @jmlopez-rod,
do you have an idea when it will be released?
